src/almanack/metrics/metrics.yml

```
@@ -52,6 +52,13 @@ metrics:
     description: >-
       Boolean value indicating the presence of a LICENSE file
       in the repository.
+  - name: "is-citable"
+    id: "SGA-GL-0005"
+    result-type: "bool"
+    result-data-key: "is-citable"
+    description: >-
+      Boolean value indicating the presence of a CITATION file
+      or some other means of citation within the repository.
```

```suggestion
      or some other means of indicating how to cite the work.
```

I'm not sure I agree with my suggestion to replace "repository" with "work", but some repos might prefer alternative citation strategies for the work and not the repo (e.g., citing pycytominer paper over the zenodo doi)

---

Thanks - I like it, let's add this in.